packages/maskbook/src/plugins/PoolTogether/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { escapeRegExp } from 'lodash-es'
+
+export const POOLTOGETHER_PLUGIN_ID = 'co.pooltogether'
+
+export const BASE_URL = 'pooltogether.com'
+export const APP_URL = 'https://app.pooltogether.com'
+export const API_URL = 'https://pooltogether-api.com/'
+export const COMMINUTY_URL = 'https://community.pooltogether.com/'
+
+export const URL_PATTERN = new RegExp(`(http|https)\:\/\/.*\.?${escapeRegExp(BASE_URL)}`)
```

Comment:
```suggestion
export const URL_PATTERN = new RegExp(`https?\:\/\/.*\.?${escapeRegExp(BASE_URL)}`)
```

Reply:
@Jack-Works can you please explain how? 
URL should match on any subdomian.

Reply:
@septs I think this will not match main domain `https://pooltogether.com`

Reply:
```suggestion
export const URL_PATTERN = new RegExp(`https?:\/\/.*\.?${escapeRegExp(BASE_URL)}`)
```